source/main/gui/panels/GUI_MainSelector.cpp

```diff
@@ -880,14 +881,25 @@ void CLASS::SetPreviewImage(Ogre::String texture)
 	}
 	catch (...)
 	{
+		Ogre::LogManager::getSingleton().stream() << "[RoR|SelectorGUI] Failed to load preview image: " << texture;
 		m_Preview->setVisible(false);
 	}
 }
 
 void CLASS::ResizePreviewImage()
 {
 	MyGUI::IntSize imgSize(0, 0);
-	Ogre::TexturePtr t = Ogre::TextureManager::getSingleton().load(m_preview_image_texture, Ogre::ResourceGroupManager::AUTODETECT_RESOURCE_GROUP_NAME);
+	Ogre::TexturePtr t;
+	try
+	{
+		t = Ogre::TextureManager::getSingleton().load(m_preview_image_texture, Ogre::ResourceGroupManager::AUTODETECT_RESOURCE_GROUP_NAME);
+	}
+	catch (...)
```

Comment:
Proposal: https://github.com/ulteq/rigs-of-rods/commit/08f1662586b12c6f6399c31c36aef7ab0daa196a